### PR TITLE
Us051 generate new enrolment code

### PR DESCRIPTION
**Pull request by andrewmil**
Added new test for generating a new enrolment code
Also updated tests which were failing because of updates to reporting-unit page

To test, run with other PR's
[response-operations-ui](https://github.com/ONSdigital/response-operations-ui/pull/85)
[ras-backstage-service](https://github.com/ONSdigital/ras-backstage/pull/71)
[rm-case-service](https://github.com/ONSdigital/rm-case-service/pull/19)

[Trello](https://trello.com/c/FIBrMsOb/72-us051-int-generate-a-new-additional-enrolment-code-l-25pts-5d)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/227115065/US051+Provide+a+new+Enrollment+Code+to+an+RU+for+a+specific+survey+over+the+phone+Large)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US051: INT Generate a new / additional enrolment code (L = 25pts = 5d)](https://trello.com/c/FIBrMsOb/72-us051-int-generate-a-new-additional-enrolment-code-l-25pts-5d)

**Comment by jwgwalton** [comment]
Can't get these working.

**Comment by tej99** [comment]
@jwgwalton same I think it's because of the response operations ui panel toggles not working as expected tho rather than the tests them selfs being wrong.

File: acceptance_tests/features/generate_new_enrolment_code.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Find Reporting Unit
```
Comment (tej99) [inline acceptance_tests/features/generate_new_enrolment_code.feature]: Generate new enrolment code?

File: acceptance_tests/features/pages/reporting_unit.py [hunk 2]
```diff
@@ -60,11 +64,9 @@ def click_change_response_status_link(ru_ref, survey, period):
 
 def get_unused_iac(ru_ref, survey_short_name):
     go_to(ru_ref)
+    click_data_panel(survey_short_name)
+    return browser.find_by_id(f'unused-enrolment-code-{survey_short_name}').value
```
Comment (tej99) [inline acceptance_tests/features/pages/reporting_unit.py]: This will fail if there isn't an unused enrolment code instead of giving a None back like it did before, not sure if you want to handle that edge case?

Comment (andrewmil) [reply acceptance_tests/features/pages/reporting_unit.py]: If it gives back None the test will still fail anyway? We could handle the error better though for sure, I'll have a look

Comment (andrewmil) [reply acceptance_tests/features/pages/reporting_unit.py]: I've added `assert enrolment_code` after calls to this function

**Comment by jwgwalton** [comment]
All working.
